src/main/webapp/feed.html

```diff
@@ -91,20 +125,82 @@ <h1>
           } else {
             let recipes = await response.json()
             recipeList.innerHTML = ""
+            clearMarkers()
             recipes.forEach((recipe) => {
               recipeList.innerHTML +=
                 '<li><a href="recipe-display.html?recipeID=' + recipe.id + '">' + recipe.title + "</a></li>"
+              let marker = createMarkerAt(recipe.location, recipe.title)
+              if (marker != null) {
+                attachInfo(recipe, marker)
+                markers.push(marker)
+              }
             })
           }
         })
       }
 
+      function createMap() {
+        map = new google.maps.Map(document.getElementById("map"), { center: { lat: 20, lng: 0 }, zoom: 1 })
+      }
+
       function getCurrentFilters() {
         var filters = Array.from(document.getElementById("selected-filters").childNodes)
         filters = filters.map((filter) => filter.id)
         return filters
       }
 
+      function clearMarkers() {
+        markers.forEach((marker) => {
+          if (marker != null) {
+            marker.setMap(null)
+          }
+        })
+        markers = []
+      }
+
+      function createMarkerAt(location, title) {
+        let marker = null
+        if (location != null && location != undefined) {
+          marker = new google.maps.Marker({
+            position: new google.maps.LatLng(location.latitude, location.longitude),
+            map: map,
+            title: title,
+          })
+        }
+        return marker
+      }
+
+      function attachInfo(recipe, marker) {
+        let contentString =
+          '<div id="content">' +
+          '<div id="siteNotice">' +
```

Comment:
this is kind of confusing, any way you can smash some of the strings together? Like `<div id="siteNotice"></div>`

Reply:
Yeah, linter puts each onto their own line, I'll try disabling that for this function.